src/AzSK/Framework/Core/SVT/Services/APIManagement.ps1

```
@@ -57,8 +57,8 @@ class APIManagement: SVTBase
     hidden [ControlResult] CheckNonARMAPIUsage([ControlResult] $controlResult)
     {
         $apimContext = New-AzureRmApiManagementContext -ResourceGroupName $this.ResourceContext.ResourceGroupName -ServiceName $this.ResourceContext.ResourceName         
```

This line may break for reader role. You need to add OnError Silently continue and check if $apimContext is null in next line